src/main/java/com/netuitive/ananke/statsd/client/request/DecrementRequest.java

```diff
@@ -27,10 +27,15 @@ public DecrementRequest withTags(List<Tag> tags) {
         return this.tags == tags ? this : new DecrementRequest(this.metric, this.value, tags, this.sampleRate);
     }
 
-    public DecrementRequest withValue(Long value) {
+    public DecrementRequest withValue(Double value) {
```

Comment:
we should override these methods with Long and then do a value.doubleValue() on it internally so we don't make any breaking changes.